src/style.css

```diff
@@ -1 +1,46 @@
-/* add styles here */
+body,
+html {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  margin-bottom: 0;
```

Comment:
Consistency with margins is important. It's recommended to add margins only to the top or bottom of elements, but not both, to avoid potential margin collapse issues.